apps/challenges/admin.py

```diff
@@ -92,6 +92,7 @@ class ChallengePhaseSplitAdmin(ImportExportTimeStampedAdmin):
         "dataset_split",
         "leaderboard",
         "visibility",
+        "leaderboard_order_is_descending"
```

Comment:
Let's change the field name in the model to `is_leaderboard_order_descending`

Reply:
@deshraj done